.github/workflows/trigger-preview.yaml

```
@@ -2,24 +2,22 @@ name: trigger-preview
 on:
   workflow_run:
     workflows:
-      - trigger-site-build
+      - trigger-book-build
```

The name here need to be consistent with whatever the PR building workflow file is called.
```suggestion
      - trigger-site-build
```